src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/with-launch-activity-name.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: withLaunchActivityName (Android SDK API)
+type: apiDoc
+shortDescription: Set the target activity name for app launch time to calculate the cold time and hot time
+tags:
+  - Mobile monitoring
+  - New Relic Mobile Android
+  - Android SDK API
+metaDescription: 'New Relic API for Android mobile app monitoring: set target activity name.'
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/withlaunchnameactivity-android-agent-api
+---
+
+## Syntax
+
+```
+NewRelic.withLaunchActivityName(string $activityName)
+```
+
+Set the target activity name for app launch time to calculate the cold time and hot time
+
+## Requirements
+
+Agent version 6.9.0 or higher.
+
+## Description
+
+By default, the New Relic Android agent calculates the cold time and hot time when the default activity loaded, this api provides the ability to customize the targe launch activity name when calculates the cold time and hot time.
```

Comment:
Capitalize API, as is done else where, or just make every instance consistent.

Reply:
fixed